addons/message-filters/style.css

```diff
@@ -0,0 +1,37 @@
+.checkboxes {
+  box-sizing: border-box;
+  padding: 2vw;
+  width: 100%;
+  background: #05f2;
+  display: block;
+  margin: 0 auto;
+  border-radius: 5px;
+  border: 1px solid #05f3;
+  width: 100%;
+  margin-top: 10px;
+}
+.checkboxes h4 {
+  padding: 5px;
+}
+.checkboxes input {
+  margin-right: 3px;
+  margin-right: 0.5vmax;
```

Comment:
There is no reason to specify multiple values for the same property. Only the last will be used.

Reply:
Whoops 😝